### PR TITLE
Static content support

### PR DESCRIPTION
**Pull request by jseppi**
ref #1337 

Static content templates (with extension `.njk`) and subfolders can be placed in the `views/content/` directory. They'll be visitable by going to `https://federalist.18f.gov/content/path/to/filename` (or with a trailing slash).

### CODE REVIEW

**Comment by jseppi** [comment]
ha, yeah, 0.2% drop :\
I'll see if I can figure out what is missing from coverage.

**Comment by jseppi** [comment]
Oh! I forgot I enabled html coverage outputs a while back. They are stored in `/coverage/`, which you can then serve with a static web server of your choice (like `python -m http.server 8000`) to view in your browser. I found some conditionals I need to add to my test cases. Coming right up.

**Comment by el-mapache** [comment]
yeah, to me 0.2% seems a bit _too_ sensitive... 🤷‍♀️

**Comment by wslack** [comment]
Endorsed; the 0.2% drop is not sufficient reason to keep from merging. We need to unblock @line47.

**Comment by jseppi** [comment]
Hold on that. Found an issue. Almost finished with fixing.

**Comment by jseppi** [comment]
Ready for re-review @el-mapache or @jmhooper

File: api/controllers/content.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  return cleaned;
+}
+
+let webpackAssets = loadAssetManifest();
+
+module.exports = {
+  serve(req, res) {
+    const reqPath = cleanRequestPath(req.path);
+
+    const availableContentFiles = getDirectoryFiles(CONTENT_PATH)
```
[the remark this answers is not in the document]
Comment (jseppi) [reply api/controllers/content.js]: I guess there would be a minor performance gain to not-pre-walk the entire directory structure, and instead just early return when a matching file is found, but given that the static content won't be that large/complex (hopefully) I don' think it matters too much.

[the remark this answers is not in the document]
Comment (el-mapache) [reply api/controllers/content.js]: yeah we can address that if serving these pages starts getting slow. I imagine that will take a pretty big file tree

[the remark this answers is not in the document]
Comment (jseppi) [reply api/controllers/content.js]: oh, I could just cache it on the first read. that's like a 2 line change.

[the remark this answers is not in the document]
Comment (el-mapache) [reply api/controllers/content.js]: Will you need a way to check if files have been added? Or will adding a file re-require this file on server reload?

[the remark this answers is not in the document]
Comment (jseppi) [reply api/controllers/content.js]: Adding a file will already trigger a re-load during dev, and I think in production we actually wouldn't want to re-load any new files once the server has started.

[the remark this answers is not in the document]
Comment (el-mapache) [reply api/controllers/content.js]: Hopefully we aren't manually adding files to the running production app 😄

File: api/controllers/content.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+let webpackAssets = loadAssetManifest();
+let availableContentFiles;
+
+module.exports = {
+  serve(req, res) {
+    const reqPath = cleanRequestPath(req.path);
+
+    if (!availableContentFiles) {
```
Comment (jseppi) [inline api/controllers/content.js]: went ahead and made this change, @el-mapache

**Pull request merged**